doc/BUILD

```
@@ -313,14 +313,22 @@ doctest(
             "source/serve/production-guide/fault-tolerance.md",
             "source/data/batch_inference.rst",
             "source/data/transforming-data.rst",
-            "source/train/faq.rst"
+            "source/train/faq.rst",
+            "source/workflows/*.rst"
         ]
     ),
     size = "large",
     tags = ["team:none"]
 )
 
 
+doctest(
+    name="doctest[workflow]",
+    files = glob(
+        include=["source/workflows/*.rst"]
```

(Not sure if you need to line brea here)
```suggestion
        include=[
            "source/workflows/**/*.rst",
            "source/workflows/**/*.md"
        ]
```